eox_hooks/actions.py

```diff
@@ -4,11 +4,20 @@
 import logging
 
 import requests
+from opaque_keys.edx.keys import CourseKey
 
 from eox_hooks.serializers import CertificateSerializer, CourseSerializer, UserSerializer
 from eox_hooks.tasks import create_enrollments_for_program
 from eox_hooks.utils import _get_course, flatten_dict, get_trigger_settings
 
+try:
```

Comment:
@mariajgrimaldi are you going to add a backend for this?

Reply:
x2

Reply:
After merging this PR